File: desert/goalkeeper.lua
loop
    b = self:findByType("ball")[1]
    p = self:findByType("peasant")
    self:command(p[1], "move", {x=18, y=b.pos.y+2})
    self:command(p[2], "move", {x=17, y=b.pos.y-1})
end

File: desert/goalkeeper.clj
(while true
  (def b (.findNearest this (.findByType this "ball")))
  (def f (.findByType this "peasant"))
  (def y (min 42 (max 29 (.y (.pos b)))))
  (.command this (first f) "move" (clj->js { :x 18, :y (+ y 2) }))
  (.command this (last f) "move" (clj->js { :x 18, :y (- y 2) })))
